### PR TITLE
hw6 done

### PR DESCRIPTION
**Pull request by b14ckster**

### CODE REVIEW

File: src/test/java/hw6/BaseClassTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    protected ActionStep actionStep;
+    protected AssertionStep assertionStep;
+
+    @BeforeClass
+    public void setUp(ITestContext testContext) throws ConfigurationException {
+        webDriver = WebDriverSingleton.getWebDriver();
+        webDriver.manage().window().maximize();
+        webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+        properties = new PropertiesConfiguration(pathToProperties);
```
Comment (akulebyakin) [inline src/test/java/hw6/BaseClassTest.java:31]: Fails here with error
```java
[ERROR] Failures: 
[ERROR]   HomePageTest>BaseClassTest.setUp:31 » Configuration Cannot locate configuratio...
[ERROR]   DifferentElementsTest>BaseClassTest.setUp:31 » Configuration Cannot locate con...
```
Cannot find configuration file
Please refactor

File: src/test/java/hw6/BaseClassTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class BaseClassTest {
+    protected WebDriver webDriver;
+    protected Configuration properties;
+    private final String pathToProperties = "src/test/resources/hw3Properties/pageData.properties";
```
Comment (akulebyakin) [inline src/test/java/hw6/BaseClassTest.java:20]: Why properties are in `hw3` folder?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+        <groupId>io.qameta.allure</groupId>
+        <artifactId>allure-testng</artifactId>
+        <version>${allure-testng.version}</version>
+        <scope>test</scope>
+    </dependency>
+<!--    <dependency>-->
+<!--        <groupId>org.slf4j</groupId>-->
+<!--        <artifactId>slf4j-simple</artifactId>-->
+<!--        <version>${slf4j.version}</version>-->
+<!--    </dependency>-->
```
Comment (akulebyakin) [inline pom.xml:93]: Let's remove commented dependency if we don't need it

File: src/main/java/hw6/pageComponents/Benefit.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+import java.util.stream.Collectors;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class Benefit extends AbstractComponent {
+
+    public Benefit(WebDriver webDriver) {
+        super(webDriver);
+    }
```
Comment (akulebyakin) [inline src/main/java/hw6/pageComponents/Benefit.java]: Сlass structures in the project are not implemented in the same style
The recommended order of class members is the following:
1. Fields
2. Constructors
3. Factory methods
4. Other Methods

Let's refactor all classes for this homework according to that structure

**Review by akulebyakin — CHANGES_REQUESTED** [review]
Please review comments

File: src/test/java/hw6/BaseClassTest.java (hunk 1 above)
Comment (b14ckster) [reply src/test/java/hw6/BaseClassTest.java:31]: added to repository

File: src/test/java/hw6/BaseClassTest.java (hunk 2 above)
Comment (b14ckster) [reply src/test/java/hw6/BaseClassTest.java:20]: This data was used in the same way in all works (starting with 3), so it was decided not to duplicate this file.

File: pom.xml (hunk 3 above)
Comment (b14ckster) [reply pom.xml:93]: removed

File: src/main/java/hw6/pageComponents/Benefit.java (hunk 4 above)
Comment (b14ckster) [reply src/main/java/hw6/pageComponents/Benefit.java]: refactored

Comment (b14ckster) [reply src/main/java/hw6/pageComponents/Benefit.java]: (but GitHub Desktop doesn't see changes, so classes refactored only local)

Comment (b14ckster) [reply src/main/java/hw6/pageComponents/Benefit.java]: done

**Review by akulebyakin — APPROVED** [review]
Approved

**Pull request merged**